clusterloader2/pkg/measurement/common/waitforpods.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
```

Comment:
Please add newline after this import. We're usually seperating k8s imports from other vendor imports.

Reply:
I have this style of import in every file `clisterloader2` files.
I agree we can change it to match k8s, but (because of many files that need to be changed) in separate PR.

Reply:
Sure.